doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,257 @@
+parameters of the model obtained after fitting. These are accessible
+through the `model_params` attribute of the IvimFit object.
+The parameters are arranged as a 4D array, corresponding to the spatial
+dimensions of the data, and the last dimension (of length 4)
+corresponding to the model parameters according to the following
+order : $\mathbf{S_{0}, f, D^*, D}$.
+"""
+
+ivimparams = ivimfit.model_params
+print("ivimparams.shape : {}".format(ivimparams.shape))
+
+"""
+As we see, we have a 20x20 slice at the height z = 27. Thus we
+have 400 voxels. We will now plot the parameters obtained from the
+fit for a voxel and also various maps for the entire slice.
+This will give us an idea about the diffusion and perfusion in
+that section. Let(i, j) denote the coordinate of the voxel. We have
+already fixed the z component as 27 and hence we will get a slice
+which is 33 units avove.
+
+"""
+
+i, j = 10, 10
+estimated_params = ivimfit.model_params[i, j, :]
+print(estimated_params)
+
+"""
+Next, we plot the results relative to the model fit.
+For this we will use the `predict` method of the IvimFit object
+to get the estimated signal.
+"""
+
+estimated_signal = ivimfit.predict(gtab)[i, j, :]
+
+plt.scatter(gtab.bvals, data_slice[i, j, :],
+            color="green", label="Actual signal")
+plt.plot(gtab.bvals, estimated_signal, color="red", label="Estimated Signal")
+plt.xlabel("bvalues")
+plt.ylabel("Signals")
+
+S0_est, f_est, D_star_est, D_est = estimated_params
+text_fit = """Estimated \n S0={:06.3f} f={:06.4f}\n
+            D*={:06.5f} D={:06.5f}""".format(S0_est, f_est, D_star_est, D_est)
+
+plt.text(0.65, 0.50, text_fit, horizontalalignment='center',
+         verticalalignment='center', transform=plt.gca().transAxes)
+plt.legend(loc='upper right')
+plt.savefig("ivim_voxel_plot.png")
+plt.close()
+"""
+.. figure:: ivim_voxel_plot.png
+   :align: center
+
+   Plot of the signal from one voxel.
+
+Now we can map the perfusion and diffusion maps for the slice. We
+will plot a heatmap showing the values using a colormap. It will be
+useful to define a plotting function for the heatmap here since we
+will use it to plot for all the IVIM parameters. We will need to specify
+the lower and upper limits for our data. For example, the perfusion
+fractions should be in the range (0,1). Similarly, the diffusion and
+pseudo-diffusion constants are much smaller than 1. We pass an argument
+called ``variable`` to out plotting function which gives the label for
+the plot.
+"""
+
+
+def plot_map(raw_data, variable, limits, filename):
+    lower, upper = limits
+    plt.title('Map for {}'.format(variable))
+    plt.imshow(raw_data.T, origin='lower', clim=(lower, upper), cmap="gray", interpolation=None)
+    plt.colorbar()
+    plt.savefig(filename)
+    plt.close()
+
+"""
+Let us get the various plots so that we can visualize them in one page
+"""
+
+plot_map(ivimparams[:, :, 0], "Predicted S0", (0, 10000), "predicted_S0.png")
```

Comment:
I think that it would be good to have attributes for each one of the model parameters, in addition to the model_params attr. 

Something like: 

    def S0_predicted(self):
        return self.model_params[..., 0]

   def perfusion_fraction(self):
       return self.model_params[..., 1]

Etc.